lib/trino-parquet/src/main/java/io/trino/parquet/reader/SimpleSliceInputStream.java

```diff
@@ -88,4 +96,21 @@ public int getByteArrayOffset()
     {
         return offset + slice.byteArrayOffset();
     }
+
+    public void ensureBytesAvailable(int bytes)
```

Comment:
Does this have any overrides? Is it pure check state and not loading any data?

Reply:
no overrides, pure check state